charts/camunda-platform/charts/optimize/templates/deployment.yaml

```diff
@@ -40,7 +40,7 @@ spec:
           - name: CAMUNDA_OPTIMIZE_IDENTITY_ISSUER_URL
             value: {{ .Values.global.identity.auth.publicIssuerUrl | quote }}
           - name: CAMUNDA_OPTIMIZE_IDENTITY_ISSUER_BACKEND_URL
-            value: "http://{{ include "common.names.dependency.fullname" (dict "chartName" "keycloak" "chartValues" . "context" $) | trunc 20 | trimSuffix "-" }}:80/auth/realms/camunda-platform"
+            value: "http://{{ include "camundaPlatform.keycloakName" . }}:80/auth/realms/camunda-platform"
```

Comment:
Feels like we could also just move the complete string inside this, and call it `camundaPlatform.issuerBackendUrl`